tools/pfe-tools/dev-server/config.ts

```diff
@@ -52,6 +53,8 @@ function renderBasic(context: Context, demos: unknown[], options: PfeDevServerCo
   return env.render('index.html', { context, options, demos });
 }
 
+const md5 = createHmac('md5', 'trust no one');
```

Comment:
Is this variable used in anything?

Reply:
Same with `join` and `stat` on line 8 / 9?

Reply:
thanks fixed. was using it to calculate etags but we don't actually need to